cmd/termui/provider/metricsProvider.go

```diff
@@ -92,22 +93,32 @@ func (smp *StatusMetricsProvider) updateMetrics() {
 	if smp.shardID != "" && smp.gatewayAddress != "" {
 		metricsURLSuffix := trieStatisticsMetricsUrlSuffix + smp.shardID
 		statusMetricsURL := smp.gatewayAddress + metricsURLSuffix
-		smp.fetchAndApplyGatewayStatusMetrics(statusMetricsURL)
+
+		if !smp.numTrieNodesSet {
+			smp.fetchAndApplyGatewayStatusMetrics(statusMetricsURL)
+		}
 	}
 }
 
 func (smp *StatusMetricsProvider) fetchAndApplyGatewayStatusMetrics(statusMetricsURL string) {
+	noErrors := true
 	numTrieNodes, err := smp.loadMetricsFromGatewayApi(statusMetricsURL)
 	if err != nil {
 		log.Info("fetch from Gateway API",
 			"path", statusMetricsURL,
 			"error", err.Error())
+		noErrors = false
```

Comment:
noErrors = false :D 
Could have been foundErrors = true to avoid double negation

Reply:
changed